sass/sass.bzl

```diff
@@ -11,219 +11,170 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"Compile Sass files to CSS"
 
-SASS_FILETYPES = FileType([
-    ".sass",
-    ".scss",
-])
+_FILETYPES = [".sass", ".scss", ".svg", ".png", ".gif"]
 
-def collect_transitive_sources(ctx):
-    source_files = depset(order="postorder")
-    for dep in ctx.attr.deps:
-        source_files += dep.transitive_sass_files
-    return source_files
+# Documentation for switching which compiler is used
+_COMPILER_ATTR_DOC = """Choose which Sass compiler binary to use.
+
+By default, we use the JavaScript-transpiled version of the
+dart-sass library, based on https://github.com/sass/dart-sass.
+This is the canonical compiler under active development by the Sass team.
+This compiler is convenient for frontend developers since it's released
+as JavaScript and can run natively in NodeJS without being locally built.
+However, it is the slowest option. If you have a substantial Sass
+codebase, consider the two other options for this attribute:
+
+1. `compiler = "@sassc"` uses the libsass compiler written in C++.
+   As of 2018, this is the most commonly used compiler.
+   It requires your Bazel setup has a working C++ compilation toolchain.
+
+   NOTE: future releases of rules_sass may remove this option, if it
+   becomes obsolete or if the maintenance burden is too high.
+
+2. Dart Sass runs the new compiler implementation natively in the Dart
+   VM. This option requires a change to the Dart Bazel rules which is
+   not yet available as of May 2018.
```

Comment:
It would be nice to link to a bug for this. @natebosch, do you know where we're tracking Bazel Dart support?

Reply:
The rules are at https://github.com/dart-lang/rules_dart 

We do not have active plans or a timeline for revisiting external bazel support.

Reply:
The readme says the rules are "in active development"... it might be a good idea to update that to give a better idea of their current status.

@jelbourn I guess the best thing to do here is link to https://github.com/dart-lang/rules_dart and say "when this is actively maintained".

Reply:
Good idea. Updated

Reply:
Done